utilities/rpp-performancetests/HOST_NEW/BatchPD_host_pkd3.cpp

```diff
@@ -3559,6 +3562,50 @@ int main(int argc, char **argv)
 
             break;
         }
+        case 81:
+        {
+            test_case_name = "color_jitter";
+
+            Rpp32f brightness[images];
+            Rpp32f contrast[images];
+            Rpp32f hue[images];
+            Rpp32f saturation[images];
+            for (i = 0; i < images; i++)
+            {
+                brightness[i] = 1.02;
+                contrast[i] = 1.1;
+                hue[i] = 0.02;
+                saturation[i] = 1.3;
+
+                // brightness[i] = 1.625;
+                // contrast[i] = 1.5;
+                // hue[i] = 0.0;
+                // saturation[i] = 1.0;
+            }
+
+            start_omp = omp_get_wtime();
+            start = clock();
+            if (ip_bitDepth == 0)
```

Comment:
you can avoid all these else cases by setting the missingFuncFlag = 1 as default